app/models/user.rb

```diff
@@ -19,12 +19,14 @@
 #  locked_at              :datetime
 #  name                   :string
 #  phone_number           :string
+#  provider               :string
 #  reset_password_sent_at :datetime
 #  reset_password_token   :string
 #  role_type              :string           not null
 #  sign_in_count          :integer          default(0), not null
 #  suspended_at           :datetime
 #  timezone               :string           default("America/New_York"), not null
+#  uid                    :string
```

Comment:
should this be `google_uid` or such? I know we're only using google for oauth but it might make it more obvious who generated this

Reply:
Yeah, agreed that uid is confusing. Makes me think the two newly-added columns (`provider` and `uid`) could be prefixed like `oauth_provider` and `oauth_uid` or `external_provider` & `external_uid`. I think `external_*` is clearer so I will try that.

Reply:
BTW one reason I'm excited to prefix the column names with _something_ is that we used to do Zendesk OAuth2 login, and we used to have a `provider` and `uid` column then, and I found them confusing in 2020.